src/math/rand/rand.go

```diff
@@ -210,6 +210,11 @@ again:
 	return f
 }
 
+// Bool returns a pseudo-random bool in [true, false].
```

Comment:
While `true` and `false` is not specified as as `1` and `0` (respectively) in Go, given the widespread use of that definition in other languages, the range should be swapped, as follows.

```diff
-[true, false].
+[false, true].
```